### PR TITLE
[Doc] Fix a few issues in the wholegraph tutorial

### PR DESCRIPTION
**Pull request by zhjwy9343**
*Issue #, if available:*

*Description of changes:*
This PR targets to fix issues in the WholeGraph tutorial. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/advanced/advanced-wholegraph.rst [hunk 1]
```diff
@@ -168,7 +168,7 @@ The output should resemble with the following screenshot which includes `NCCL IN
...
+Different from typical GraphStorm training where node features are stored in the ``node_feat.dgl`` in each partition, to utilize WholeGraph, the node features must be converted into a compatible WholeGraph format. After construct your raw data into GraphStorm required DistDGL format following this :ref:`graph construction tutorial<use-own-data>`, users can achieve this by utilizing the `convert_feat_to_wholegraph.py <https://github.com/awslabs/graphstorm/blob/main/tools/convert_feat_to_wholegraph.py>`_ script with ``--dataset-path`` pointing to the folder with the partitioned graph data and specify the features to be transformed using the ``--feat-names`` option. For example:
```
Comment (zheng-da) [inline docs/source/advanced/advanced-wholegraph.rst:171]: change `--feat-names` to `--node-feat-names`. also change the argument name in the command below.

**Comment by zheng-da** [comment]
in the place where we need to build docker containers, we need to explicitly that users can only build one docker container image and copy the image to other machines.

File: docs/source/advanced/advanced-wholegraph.rst (hunk 1 above)
Comment (zhjwy9343) [reply docs/source/advanced/advanced-wholegraph.rst]: Changed.

**Comment by zhjwy9343** [comment]
Add the one docker image requirement to both Distributed Cluster and WholeGraph toturial.

**Comment by zhjwy9343** [comment]
> in the place where we need to build docker containers, we need to explicitly that users can only build one docker container image and copy the image to other machines.

Please check the newly added contents.

**Comment by classicsong** [comment]
Shall we merge it into 0.2.1?

**Comment by zhjwy9343** [comment]
> Shall we merge it into 0.2.1?

Yes. It does not impact our code and pip package.

File: docs/source/scale/distributed.rst [hunk 2]
```diff
@@ -11,12 +11,18 @@ GraphStorm can scale to the enterprise-level graphs in the distributed mode by u
...
+A cluster contains several instances each of which can run GraphStorm Docker container.
+
+To create such a cluster, in one instance, please follow the :ref:`Environment Setup <setup_docker>` description to setup GraphStorm Docker container environment, and use a Docker management system, e.g., AWS ECR, to upload the Docker image built in the instance to a Docker repository and pull it to the rest of the instances in the cluster. 
+
+If there is no such Docker manangement system available in your environment, in **each** instance of the cluster, follow the :ref:`Environment Setup <setup_docker>` description to build a GraphStorm Docker image, and start the image as a container. Then exchange one key from inside of one GraphStorm Docker containers to the rest containers in the cluster, i.e., copy the keys from the ``/root/.ssh/id_rsa.pub`` from one container to ``/root/.ssh/authorized_keys`` in containers on all other containers.
```
Comment (classicsong) [inline docs/source/scale/distributed.rst:25]:
```suggestion
If there is no such Docker manangement system available in your environment, in **each** instance of the cluster, follow the :ref:`Environment Setup <setup_docker>` description to build a GraphStorm Docker image, and start the image as a container. Then exchange the ssh key from inside of one GraphStorm Docker containers to the rest containers in the cluster, i.e., copy the keys from the ``/root/.ssh/id_rsa.pub`` from one container to ``/root/.ssh/authorized_keys`` in containers on all other containers.
```

File: docs/source/scale/distributed.rst [hunk 3]
```diff
@@ -75,7 +81,6 @@ In the container environment, users can check the connectivity with the command
 If succeeds, you should login to the container in the ``<ip-in-the-cluster>`` instance. 
 
 If not, please make sure there is no limitation of port 2222.
```
Comment (classicsong) [inline docs/source/scale/distributed.rst:83]:
```suggestion
If not, please make sure there is no restriction of exposing port 2222. 
```

**Comment by zhjwy9343** [comment]
Thanks. accepted the two new revisions.

**Review by classicsong — APPROVED** [review]

**Pull request merged**